modules/rules/cr.js

```diff
@@ -73,24 +78,67 @@ class CR {
                     newEntry.push(word);
                 }
             }
-            crEntries[number.replace(/\.$/, "")] = this.highlightRules(newEntry.join(" "));
+            entry = this.highlightRules(newEntry.join(" "));
+
+            crEntries[number] = '';
+            entry.split('\n').forEach(line => {
+                if (line.match(/^Example: /i)) {
+                    if (!crEntries[number+' ex']) crEntries[number+' ex'] = '';
+                    crEntries[number+' ex'] += line.replace(/^Example: /i, '**Example:** ') + '\n\n';
+                } else {
+                    crEntries[number] += line + '\n';
+                }
+            })
         }
         return crEntries;
     }
 
     highlightRules(text) {
-        return text.replace(/rule \d{3}\.\w*\.?/g, "**$&**");
+        return text.replace(/rule \d{3}\.\w*\.?/ig, "**$&**");
+    }
+
+    appendSubrules(parameter) {
+        let description = this.crData[parameter];
+        if (description && this.crData[parameter + 'a']) {
+            let x = 97; // 'a'
+            while(this.crData[parameter + String.fromCharCode(x)]) {
```

Comment:
Neat idea, although turning this into a `for` loop made it unintelligible enough to warrant an extra comment line.